src/main/java/gameframework/gui/GameWindow.java

```diff
@@ -17,17 +17,18 @@
 	protected final ObservableValue<Integer> score;
 	protected final ObservableValue<Integer> life;
 
-	public GameWindow(GameCanvas gameCanvas, GameConfiguration configuration,
-			ObservableValue<Integer> score, ObservableValue<Integer> life) {
+	public GameWindow(String gameName, GameCanvas gameCanvas,
+			GameConfiguration configuration, ObservableValue<Integer> score,
+			ObservableValue<Integer> life) {
 		if (gameCanvas == null) {
 			throw new IllegalArgumentException("gameCanvas is null");
 		}
 		this.score = score;
 		this.life = life;
-		this.frame = new Frame("Default Game");
+		this.frame = new Frame(gameName);
 		this.gameCanvas = gameCanvas;
-		this.gameCanvas.setSize(//
```

Comment:
these empty comments help the Java automatic code formatter (Alt+Shift+f). Please keep them.